src/domain/constants.js

```diff
@@ -183,6 +183,21 @@ export const FILES_VIEWER_DISPLAY = {
   list: 'list'
 };
 
+export const DOCUMENT_REQUEST_TYPE = {
+  read: 'read',
+  write: 'write'
+};
+
+export const DAY_OF_WEEK = {
+  monday: 'Mon',
+  tuesday: 'Tue',
+  wednesday: 'Wed',
+  thursday: 'The third day!',
```

Comment:
As much as I like it personally, I think for the sake of anyone else potentially dealing with the code at some time, we should not make them scratch their head too much (or make them laugh out loud too much) :rofl:

Reply:
Hehe, this was an Easter egg just for you 😏 I was going to remove it before merging.

Reply:
Sneaky :wink: